.github/workflows/tests-karma-dashboard.yml

```
@@ -61,6 +71,26 @@ jobs:
       - name: Increase max number of file watchers
         run: echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
 
+      - name: Get Chromium executable path
+        id: chromium_path
+        run: |
+          CHROMIUM=$(node -p "const puppeteer = require('puppeteer'); puppeteer.executablePath();")
+          echo "::set-output name=chromium_path::${CHROMIUM}"
+
+      - name: Run dashboard integration tests with Percy
+        run: |
+          npx percy exec --parallel --config=percy.config.karma.yml -- \
+            npm run test:karma:dashboard -- --headless --viewport=1600:1000 --coverage --shard=$SHARD || \
+          npx percy exec --parallel --config=percy.config.karma.yml -- \
+            npm run test:karma:dashboard:retry-failed -- --headless --viewport=1600:1000 --shard=$SHARD
+        env:
+          DISABLE_ERROR_BOUNDARIES: true
+          SHARD: ${{ matrix.shard }}
+          PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
```

```suggestion
          PERCY_TOKEN: ${{ secrets.PERCY_TOKEN_DASHBOARD }}
```

This will send dashboard karma snapshots to a new project that I've just set up on Percy.